debian/copyright

```diff
@@ -0,0 +1,37 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+
+Files: *
+Copyright: 2022 Nicolai Buchwitz <n.buchwitz@kunbus.com>
```

Comment:
Copyright is Kunbus GmbH
You are "just" the Author

Reply:
I used that with the maya package:

Copyright: KUNBUS GmbH